### DISCUSSION TITLE
Meeting 1: Kick-off meeting (30th November 14:00 UTC)

### DISCUSSION THREAD

**Discussion by ml-evs** [marda-alliance/metadata_extractors#4] [Announcements]
Our first meeting will run on [November 30th at 14:00 UTC/15:00 CET/09:00 EST](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20221130T14) on Jitsi at https://meet.jit.si/marda_extractors.

We will begin with a quick presentation of the current ideas with some surveys to gauge the level of engagement, then we will move into Q&A and open discussions.

All are welcome!

**Comment by dwinston** [comment]
oh shoot, I had this on my calendar as 9 PST / 12 EST. 😵‍💫

**Reply by ml-evs** [reply]
Ah, that's a shame @dwinston! We had a decent turnout in the end so at least it looks like there will be follow-up meetings!

I've got a recording of the intro talk that I will share over email (and above) once the captions are sorted, and we also took minutes. I briefly discussed with Ken about tie-ins with the data dictionaries WG, so feel free to advertise/share anything regarding that group here!

**Comment by ml-evs** [comment]
Peter Kraus has kindly entered the minutes for meeting over at https://github.com/marda-alliance/metadata_extractors/blob/main/meetings/1-meeting-30-11-2022/minutes.md.

I have also added a link to the talk recording (https://www.youtube.com/watch?v=6x5Ow-CLRWg) and slides (https://docs.google.com/presentation/d/1QTqDEO3H1s_wAtcE9DD6k1eCJ7GzvdV2spjHE94bEQw/edit?usp=sharing).
